collector/src/test/java/org/apache/hertzbeat/collector/collect/redfish/RedfishCollectImplTest.java

```diff
@@ -77,88 +79,96 @@ void mockCollect() throws Exception {
         CollectRep.MetricsData.Builder builder = CollectRep.MetricsData.newBuilder();
         List<String> aliasField = new ArrayList<>();
         aliasField.add("$.['@odata.id']");
-        redfishProtocol.setSchema("/redfish/v1/Chassis/{ChassisId}/PowerSubsystem/PowerSupplies");
+        redfishProtocol.setSchema("/redfish/v1/chassis/{ChassisId}/PowerSubsystem/PowerSupplies");
```

Comment:
Hi, redifshProtocol schema field means the Resource collection URI which you want to monitor. Users can refer to [DSP0268](https://www.dmtf.org/sites/default/files/standards/documents/DSP0268_2023.3.pdf) Section 5.2 content to find the according Resource collection URI. So in this test, we want to monitor PowerSupplies in PowerSupplyCollection. Refer to the table, the according URI is "/redfish/v1/Chassis/{ChassisId}/PowerSubsystem/PowerSupplies". So I think "Chassis" with capital “C” is correct, misspelled test cases may confuse developers. Sorry for not providing detailed documentation explaining usage details. I will impove the redfish collect and complete documentation soon.

Reply:
Thanks for the clarification, I've reverted.

Now just change the variable name: `String Chassis` -> `String chassis`.